ecs-cli/modules/utils/waiters/waiters.go

```diff
@@ -46,27 +46,36 @@ const (
 // and returns a bool to stop the wait or error if something unexpected happens
 type waiterAction func(retryCount int) (bool, error)
 
-// waitUntilComplete executes the waiterAction for maxRetries number of times, waiting for delayWait time between execution
-func WaitUntilComplete(action waiterAction, entity entity.ProjectEntity, timeoutMessage string, isService bool) error {
```

Comment:
A function with the same name `WaitUntilComplete` is being used in cloudformation/client.go. However, it is not actually the same function. My IDE shows that this `WaitUntilComplete` is defined cloudformation/client.go:202